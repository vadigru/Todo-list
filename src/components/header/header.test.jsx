import React from "react";
import renderer from "react-test-renderer";

import Header from "./header";

it(`Should render Header component`, () => {
  const tasks = [
    {
      task: `Task 1`,
      isActive: true
    },
    {
      task: `Task 2`,
      isActive: true
    },
  ];

  const tree = renderer
    .create(
        <Header
          tasks={tasks}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
