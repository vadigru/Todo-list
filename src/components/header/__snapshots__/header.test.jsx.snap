// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header component 1`] = `
<header
  className="header"
>
  <h1
    className="header__title"
  >
    TODO
  </h1>
  <h2
    className="header__tasks-count"
  >
    You have 
    2
     active task/s
  </h2>
</header>
`;
